sdk/storage/storage-file-share/src/Clients.ts

```diff
@@ -1410,16 +1410,16 @@ export class ShareClient extends StorageClient {
     }
   }
 
-  /**
-   * Get a {@link ShareLeaseClient} that manages leases on the share.
-   *
-   * @param {string} [proposeLeaseId] Initial proposed lease Id.
-   * @returns {ShareLeaseClient} A new ShareLeaseClient object for managing leases on the share.
-   * @memberof ShareClient
-   */
-  public getShareLeaseClient(proposeLeaseId?: string) {
-    return new ShareLeaseClient(this, proposeLeaseId);
-  }
+  // /**
```

Comment:
I think it's cleaner to remove the code instead of commenting.  In the future it is easy to bring them back using `git revert`

Reply:
Done.